cristin-commons/src/main/java/no/unit/nva/utils/UriUtils.java

```diff
@@ -128,4 +132,32 @@ public static URI createNvaPositionId(String code) {
         return attempt(() -> new URI(positionBase.getScheme(), positionBase.getHost(), positionBase.getPath(), code))
                 .orElseThrow();
     }
+
+    /**
+     * Convenience method to encode a uri.
```

Comment:
to encode or decode?

Reply:
To decode or not to decode - that's the question...